admin/includes/functions/general.php

```diff
@@ -2135,6 +2135,43 @@ function zen_products_id_valid($products_id) {
     }
   }
 
+
+/*
+ *  Check if option name is not expected to have an option value (ie. text field, or File upload field)
+ */
+  function zen_option_name_no_values($option_name_id) {
```

Comment:
Something is better than what was "quick" generated.  Was trying to keep short, but also not specific to the attribute controller and still make sense for other uses. (May be desired/needed to have an option value as part of the submission process to the shopping cart.  May just need to be a name that when questioned is evaluated by looking at the code/notes.)